lib/sles4sap_publiccloud.pm

```diff
@@ -434,6 +434,7 @@ sub stop_hana {
     else {
         my $sapadmin = lc(get_required_var('INSTANCE_SID')) . 'adm';
         $self->run_cmd(cmd => $cmd, runas => $sapadmin, timeout => $timeout);
+        $self->{my_instance}->wait_for_ssh(username => 'cloudadmin', scan_ssh_host_key => 1);
```

Comment:
Is this one intended to add a `wait_for_ssh` in `stop` and `kill` scenario?
What is the purpose of this change?